evm_loader/program/src/evm/buffer.rs

```diff
@@ -154,8 +143,7 @@ impl serde::Serialize for Buffer {
         use serde::ser::SerializeStructVariant;
 
         match &self.inner {
-            Inner::Empty => serializer.serialize_unit_variant("evm_buffer", 0, "empty"),
-            Inner::Owned { data } => {
+            Inner::Owned(data) => {
                 let bytes = serde_bytes::Bytes::new(data);
                 serializer.serialize_newtype_variant("evm_buffer", 1, "owned", bytes)
```

Comment:
I think the count can be changed from `0` without breaking changes, because from what I know, NeonEVM does not execute iterative transactions across NeonEVM versions, so one NeonEVM version starts and finishes all iterative transactions before updating it to a new version. But @anton-lisanin knows best.

Reply:
Actually what should the integer value be. Do we have tests covering ser/de?

Reply:
See https://github.com/neonlabsorg/neon-evm/pull/261.